<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="text_for_cipher">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_for_decrypted">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>371</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите текст для зашифровки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите текст для дешифровки</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cipher_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypted_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Дешифровать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decrypted_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cipher_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cipher_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>encrypt_text()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decrypted_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>decrypt_text()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>encrypt_text()</slot>
  <slot>decrypt_text()</slot>
 </slots>
</ui>
